ci-operator/step-registry/ipi/conf/gcp/xpn/ipi-conf-gcp-xpn-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH=/tmp/install-config-xpn.yaml.patch
+
+BASE_DOMAIN="installer.gcpxpn.devcluster.openshift.com"
```

Comment:
installer.gcpxpn.devcluster.openshift.com is in the service project we use for local testing (openshift-installer-shared-vpc). The service project in this case, for ci, is openshift-gce-devel. The host project in both cases is openshift-dev-installer.